content/en/docs/demo/_index.md

```diff
@@ -57,19 +56,15 @@ found here:
 - [Image Provider Service](services/image-provider/)
 - [React Native App](services/react-native-app/)
 
-## Scenarios
+## Feature Flag Scenarios
 
-How can you solve problems with OpenTelemetry? These scenarios walk you through
-some pre-configured problems and show you how to interpret OpenTelemetry data to
+How can you solve problems with OpenTelemetry? These
+[feature flag enabled scenarios](feature-flags/) walk you through some
+pre-configured problems and show you how to interpret OpenTelemetry data to
 solve them.
 
 We'll be adding more scenarios over time.
```

Comment:
I'd drop this line as we have no concrete plans for this.

Reply:
Thanks @mviitane! Updated 😄